### DISCUSSION TITLE
Pointing ruby-lsp to a custom Gemfile

### DISCUSSION THREAD

**Discussion by erikastr** [zed-industries/zed#20772] [Feature Requests]
Hello,

There is currently a way in VSCode to specify a custom Gemfile for Ruby LSP when working on a project with a Ruby version older than `>= 3.0`

https://github.com/Shopify/ruby-lsp/tree/main/vscode#using-a-custom-gemfile

Is it possible to achieve something similar in Zed? From my understanding, Zed looks for ruby-lsp in `$PATH` by default.

**Comment by floptwo** [comment]
It's not exactly about the gemfile but I manage to make it work with the binstub the same way it's explained for solargraph.
https://gist.github.com/floptwo/fdb2df57e7e7cde3781e64de0007f5a9

Maybe juste the documentation need to be updated ?

**Reply by erikastr** [reply]
Nice, this works perfectly!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
